components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/resources/applications.yaml

```diff
@@ -1790,7 +1785,7 @@ paths:
       tags:
         - Application Metadata
       summary: |
-        Retrieve all the metadata related to the auth protocol WS_Trust
+        Retrieve all the metadata related to the auth protocol WS Trust
```

Comment:
Is missing full stop is intentional?

Reply:
Yes. We decided not to include full stops in the summary fields